doc/src/manual/embedding.md

```diff
@@ -13,6 +13,7 @@ We start with a simple C program that initializes Julia and calls some Julia cod
 
 ```c
 #include <julia.h>
+JULIA_DEFINE_FAST_TLS() // only define this once, in an executable (not in a shared library) if you want fast code.
```

Comment:
Of course. The point of this being an opaque macro though is that you can just blindly use it and depend on it doing the right thing. Almost worth including this in v0.6 backports just for that convenience, but since this commit is only about ergonomics, it's not essential.